data/json/npcs/TALK_COMMON_MISSION.json

```diff
@@ -39,24 +39,25 @@
     "type": "talk_topic",
     "category": "CATEGORY_MISSION",
     "dynamic_line": {
-      "npc_has_mission": true,
-      "none": {
-        "u_has_mission": true,
-        "none": "I don't have any jobs for you.",
-        "one": "I don't have any more jobs for you.",
-        "many": "I don't have any more jobs for you."
+      "has_no_available_mission": {
+        "has_no_assigned_mission": "I don't have any jobs for you.",
+        "no": { "has_many_assigned_missions": "I don't have any more jobs for you.", "no": "I don't have any more jobs for you." }
       },
-      "one": {
-        "u_has_mission": true,
-        "none": "I have another job for you.  Want to hear about it?",
-        "one": "I just have one job for you.  Want to hear about it?",
-        "many": "I just have one job for you.  Want to hear about it?"
-      },
-      "many": {
-        "u_has_mission": true,
-        "none": "I have other jobs for you.  Want to hear about them?",
-        "one": "I have other jobs for you.  Want to hear about them?",
-        "many": "I have more jobs for you.  Want to hear about them?"
+      "no": {
+        "has_many_available_missions": {
+          "has_no_assigned_mission": "I have other jobs for you.  Want to hear about them?",
+          "no": {
+            "has_many_assigned_missions": "I have more jobs for you.  Want to hear about them?",
+            "no": "I have other jobs for you.  Want to hear about them?"
+          }
+        },
+        "no": {
+          "has_no_assigned_mission": "I have another job for you.  Want to hear about it?",
+          "no": {
+            "has_many_assigned_missions": "I just have one job for you.  Want to hear about it?",
+            "no": "I just have one job for you.  Want to hear about it?"
+          }
+        }
```

Comment:
I think the nesting in this is a little confusing, and when to use "some", "another", "more", "other", etc are a bit unclear.  Also, it doesn't seem like it should matter if the player already has multiple missions assigned.  What about the following?
```json
  {
    "id": "TALK_MISSION_LIST",
    "type": "talk_topic",
    "category": "CATEGORY_MISSION",
    "dynamic_line": {
      "has_no_assigned_mission": {
        "has_no_available_mission": "I don't have any jobs for you.",
        "no": {
          "has_many_available_missions": "I have some jobs for you.  Want to hear about them?",
          "no": "I just have one job for you.  Want to hear about it?"
        }
      },
      "no": {
        "has_no_available_mission": "I don't have any more jobs for you.",
        "no": {
          "has_many_available_missions": "I have more jobs for you.  Want to hear about them?",
          "no": "I have another job for you.  Want to hear about it?"
        }
      }
    },
    "responses": [
      { "text": "Oh, okay.", "topic": "TALK_NONE", "condition": "has_no_available_mission" },
      {
        "text": "Never mind, I'm not interested.",
        "topic": "TALK_NONE",
        "condition": { "not": "has_no_available_mission" }
      }
    ]
  },
```